pages/_app.js

```diff
@@ -10,21 +10,25 @@ const Loading = () => <Fragment>Loading...</Fragment>;
 const MyApp = ({ Component, pageProps, router }) => {
   const [manualEnabled, setManualEnabled] = useState(false);
   const [isAnimationOn, setIsAnimationOn] = useState(true);
+
   useEffect(() => {
     if (manualEnabled) {
       setManualEnabled(manualEnabled);
     }
-
     if (isAnimationOn) {
       setIsAnimationOn(isAnimationOn);
     }
   }, []);
+  
+  const clientHintDeviceMemory = pageProps.clientHintDeviceMemory;
+  console.log('_app.js => MyApp => clientHintDeviceMemory: ', clientHintDeviceMemory);
 
+  let animationAllowed = true;
   const memoryStatus = useMemoryStatus();
   if (!memoryStatus) return <Loading />;
-  const { overLoaded } = memoryStatus;
 
-  let animationAllowed = true;
+  const { overLoaded } = clientHintDeviceMemory ? false : memoryStatus;
```

Comment:
Does not make sense and can we use false to set value to object element (overLoaded)?